x-pack/docs/en/security/authentication/jwt-realm.asciidoc

```diff
@@ -435,11 +440,40 @@ and {es} used the `jwt_role1` role:
 ----
 {"username":"user123_runas","roles":["jwt_role1"],"full_name":null,"email":null,"metadata":{},
 "enabled":true,"authentication_realm":{"name":"jwt2","type":"jwt"},"lookup_realm":{"name":"native",
-"type":"native"},"authentication_type":"realm"}% 
+"type":"native"},"authentication_type":"realm"}%
 ----
 
+[[jwt-realm-jwkset-reloading]]
+===== PKC JWKS reloading
+JWT authentication supports signature verification using PKC (Public Key Cryptography) or HMAC algorithms.
+PKC JWKS can contain public RSA and EC keys. HMAC JWKS or HMAC UTF-8 JWK contain secret keys.
+JWT issuers typically rotate PKC JWKs more frequently (ex: daily), because RSA and EC public keys are
+designed to be easier to distribute than secret keys like HMAC.
+
+JWT realms load a PKC JWKS, and a HMAC JWKS or HMAC UTF-8 JWK, at startup.
+JWT realms can also reload PKC JWKS contents at runtime; reload is triggered by signature validation failures.
+
+CAUTION: HMAC JWKS or HMAC UTF-8 JWK reloading is not supported at this time.
```

Comment:
```suggestion
NOTE: HMAC JWKS or HMAC UTF-8 JWK reloading is not supported at this time.
```

Reply:
I think that this can just be a `NOTE`. A `CAUTION` admonition is typically reserved for severe instances, such as when data loss can occur.